### ISSUE TITLE
add support to :fullscreen

### ISSUE THREAD

**Issue by yukulele** [LeaVerou/prefixfree#112]
with fullscreen API, their is a new <tt>:fullscreen</tt> pseudo class.

however, webkit and mozilla use prefixed pseudo class with dash : 
- <tt>-moz-full-screen</tt>
- <tt>-webkit-full-screen</tt>

https://developer.mozilla.org/en-US/docs/CSS/:fullscreen

actually, -prefix-free ignore <tt>:fullscreen</tt> and <tt>:full-screen</tt>

**Comment by LeaVerou** [comment]
Yup, this needs to be added. Although the spec is still very draft, but oh well.

**Comment by larsenwork** [comment]
would be nice if this was added :+1:

**Comment by LeaVerou** [comment]
@yukulele ?

**Comment by yukulele** [comment]
yes!

**Comment by LeaVerou** [comment]
Um, why did you close the issue?

**Comment by yukulele** [comment]
Sorry, i did close some of my oldest issues. 😕
Is prefix-free dev still active?
